.changeset/tidy-melons-promise.md

```diff
@@ -0,0 +1,7 @@
+---
+'@marigold/components': major
+'@marigold/theme-b2b': major
+'@marigold/docs': major
+---
+
+Introducing Calendar component
```

Comment:
oh I think you created two changesets :D

Reply:
Deleted